lib/remote/httpserverconnection.cpp

```diff
@@ -541,13 +543,16 @@ void HttpServerConnection::ProcessMessages(boost::asio::yield_context yc)
 
 			Log logMsg (LogInformation, "HttpServerConnection");
 
-			logMsg << "Request: " << request.method_string() << ' ' << request.target()
+			logMsg << "Request " << request.method_string() << ' ' << request.target()
 				<< " (from " << m_PeerAddress
 				<< "), user: " << (authenticatedUser ? authenticatedUser->GetName() : "<unauthenticated>")
 				<< ", agent: " << request[http::field::user_agent]; //operator[] - Returns the value for a field, or "" if it does not exist.
 
-			Defer addRespCode ([&response, &logMsg]() {
-				logMsg << ", status: " << response.result() << ").";
+			Defer addRespCode ([&response, start, &logMsg]() {
+				using namespace std::chrono;
```

Comment:
Please avoid `using namespace` as this can cause problems when new names are added to a namespace. [[1]](https://google.github.io/styleguide/cppguide.html#Namespaces) [[2]](https://stackoverflow.com/q/1452721)

Reply:
I know. And I did it anyway because of the little scope and its many usages of that lengthy namespace.

Reply:
As you say yourself, it's only a little scope, so there are not that many usages. It would be absolutely no problem to just spell it out the tree times. And if the namespace name was really that long, you could also define a shorthand for it. All avoiding the problems that might occur if names are added to the namespace.

And if you already know better, why do I have to waste my time writing this comment?